chart/templates/ssh-proxy.yaml

```diff
@@ -0,0 +1,135 @@
+{{- $component := "ssh-proxy" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ mustMerge (dict "name" $component) . | include "eirinix.component-name" }}
+  labels:
+    app.kubernetes.io/component: {{ $component }}
+{{ include "eirinix.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "eirinix.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: {{ $component }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "eirinix.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: {{ $component }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ .Values.service_account.name | quote }}
+      initContainers:
+      - name: generate-config
+        {{- with .Values.image }}
+        image: {{ printf "%s/eirinix-%s-setup:%s" .repository $component .tag | quote }}
+        {{- end }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: CC_API_URL
+          value: {{ index .Values.ssh "cc-api-url" }}
+        - name: UAA_TOKEN_URL
+          value: {{ index .Values.ssh "uaa-token-url" }}
+        volumeMounts:
+        - name: config
+          mountPath: /run/secrets/config
+        - name: uaa-client
+          mountPath: /run/secrets/uaa-client-password
+          subPath: password
+        - name: ssh-proxy-host-key
+          mountPath: /run/secrets/ssh-proxy-host-key.key
+          subPath: proxy.key
+      containers:
+      - name: {{ $component }}
+        {{- with .Values.image }}
+        image: {{ printf "%s/eirinix-%s:%s" .repository $component .tag | quote}}
+        {{- end }}
+        command: [ /{{ $component }} ]
+        args:
+        - --config
+        - /run/secrets/config/eirini-ssh-proxy.json
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: SSH_PROXY_KUBERNETES_NAMESPACE
+          value: {{ .Values.namespace }}
+        - name: SSH_PROXY_DAEMON_PORT
+          value: "2222"
+        volumeMounts:
+        - name: config
+          mountPath: /run/secrets/config
+        - name: uaa-ca-cert
+          mountPath: /run/secrets/uaa-ca.crt
+          subPath: ca.crt
+        - name: cc-api-ca-cert
+          mountPath: /run/secrets/cc-api-ca.crt
+          subPath: ca.crt
+        resources:
+          {{- index .Values.resources $component | toYaml | nindent 12 }}
+      volumes:
+      - name: config
+        emptyDir: {}
+      - name: uaa-client
+        secret:
+          secretName: var-uaa-clients-ssh-proxy-secret
+          items:
+          - key: password
+            path: password
+      - name: ssh-proxy-host-key
+        secret:
+          secretName: var-eirini-ssh-proxy-key
```

Comment:
We have to make those configurable too, I've just tested with KubeCF with Native Eirini, and this is not here anymore. `var-diego-ssh-proxy-host-key`  I think was the correct one